episodes/20-preparing.md

```diff
@@ -31,6 +31,7 @@ A teaching plan outlines the structure of your teaching session, including detai
 - welcome, introductions and motivation for the lesson - to introduce yourself, aims and objectives of the lesson and how it will help learners
 - setup - to check if everyone is ready to proceed with the lesson
 - segments of teaching and exercises from the lesson and any other planned activities (e.g. group discussions, wrap-up and feedback)
+- checkpoints - places where you have planed to stop to check-in with leaners. Checkpoints are especially useful at the beginning of a lesson when you may not have as many exercises but the learners need to have the data downloaded or tool opened to be able to follow along.
```

Comment:
```suggestion
- checkpoints - places where you have planned to stop to check-in with learners. Checkpoints are especially useful at the beginning of a lesson when you may not have as many exercises but the learners need to have the data downloaded or tool opened to be able to follow along.
```

Reply:
lol I wonder how often I accidentally type "leaners" it isn't the first time for sure